plugin/src/sidebar.ts

```diff
@@ -35,9 +33,17 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
       },
     });
     reearth.clientStorage.setAsync("overrides", payload);
-    if (!doNotShowWelcome) {
-      reearth.modal.show(welcomeScreenHtml, { background: "#000000bf" });
-    }
+    reearth.clientStorage.getAsync("doNotShowWelcome").then((value: any) => {
+      if (!value && !reearth.scene.inEditor) {
+        reearth.modal.show(welcomeScreenHtml, {
+          width: reearth.viewport.width,
+          height: reearth.viewport.height,
+        });
+        welcomPagIsOpen = true;
+      }
+    });
+  } else if (action === "storageSaveWelcomeScreen") {
+    reearth.clientStorage.setAsync(payload.key, payload.value);
```

Comment:
This can just use the `storageSave` action

Reply:
done

Reply:
6dc6f5efc904b2988a326ea6674b939ecdca1183